source/fs/fsutil.c

```diff
@@ -525,11 +525,9 @@ bool PathCopyVrtToFat(char* dest, char* orig, u32* flags) {
         }
     }
     
-    // check destination write permission (special paths only)
-    if (((*dest == '1') || (strncmp(dest, "0:/Nintendo 3DS", 16) == 0)) &&
-        (!flags || (*flags & (OVERWRITE_CUR|OVERWRITE_ALL))) &&
-        (!flags || !(*flags & OVERRIDE_PERM)) && 
-        !CheckWritePermissions(dest)) return false;
+    // check destination write permission
+    if ((!flags || !(*flags & OVERRIDE_PERM)) && !CheckWritePermissions(dest))
+        return false;
     
```

Comment:
I only intended to do pull requests for this commit and the previous one. That's why i had them in two seperate "pr-" branches.

I moved the permission check from PathCopy into the worker, because i failed to realize it will be called for every single file when copying a folder. When copying single files (which is the only thing it tested) this saved a permission check in most cases.